### PR TITLE
#33: Updated code documentation

### PR DESCRIPTION
**Pull request by bonkey**
#33: Updated code documentation

### CODE REVIEW

File: AtlasSDK/AtlasSDK/API/Options.swift [hunk 1]
```diff
@@ -4,35 +4,72 @@
...
     public let useSandboxEnvironment: Bool
+
+    /// Sets language of the interface. Overrides sales channel default language.
     public let interfaceLanguage: String?
+
+    /// Sets config service URL. In regular cases it shouldn't be changed.
     public let configurationURL: URL
-    public let displayRecommendations: Bool
 
+    /// Sets if recommendations are handled by the host app.
```
Comment (hani-ibrahim) [inline AtlasSDK/AtlasSDK/API/Options.swift]: `Sets if recommendations are displayed at the order confirmation page and handled by the host app.`?

File: AtlasSDK/AtlasSDK/Notifications.swift [hunk 2]
```diff
@@ -4,24 +4,36 @@
...
+
+    /**
+     Posted on `AtlasAPI` deauthorization.
+     - SeeAlso: `AtlasAPI.deauthorize(with:)`
+     */
     public static let AtlasDeauthorized = NSNotification.Name(rawValue: "Atlas.NotificationDeauthorized")
+
+    /**
+     Posted on `AtlasAPI` authorization state change.
+     - SeeAlso: `AtlasAPI.authorize(with:)`
```
Comment (hani-ibrahim) [inline AtlasSDK/AtlasSDK/Notifications.swift]: we can add
>  - SeeAlso: `AtlasAPI.deauthorize(with:)`

**Comment by hani-ibrahim** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

File: AtlasSDK/AtlasSDK/API/Options.swift (hunk 1 above)
Comment (bonkey) [inline AtlasSDK/AtlasSDK/API/Options.swift]: I didn't want to refer to visual behaviour in API module...

Comment (hani-ibrahim) [inline AtlasSDK/AtlasSDK/API/Options.swift]: that's a good point. But then may be we can say that this option is used only for AtlasUI to be clear

**Comment by bonkey** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

**Pull request merged**